host/app/components/schema-inspector.gts

```diff
@@ -103,9 +134,41 @@ export default class SchemaInspector extends Component<{ Args: { module: Record<
 
   @action
   create() {
-    if (!this.selected) {
+    if (!this.selectedCard) {
       return;
     }
     this.showEditor = true;
   }
+
+}
+
+function fieldName([fieldName, ]: [fieldName: string, fieldDefinition: FieldDefinition]): string {
+  return fieldName;
+}
+
+function fieldType([_fieldName, fieldDefinition ]: [fieldName: string, fieldDefinition: FieldDefinition]): FieldType {
+  return fieldDefinition.type;
+}
+
+function fieldCard([_fieldName, fieldDefinition ]: [fieldName: string, fieldDefinition: FieldDefinition]): CardReference {
+  return fieldDefinition.card;
+}
+
+function externalCardName(ref: ExternalReference): string {
+  if (ref.name === 'default') {
+    return `${ref.module} card`;
+  }
+
+  return `'${ref.name}' card of ${ref.module}`;
+}
+
+function getCardPath(ref: ExternalReference, currentPath: string): string | undefined {
+  if (ref.module.startsWith('.') || ref.module.startsWith('/')) {
+    let pathSegments = currentPath.split('/');
+    pathSegments.pop();
+
+    let url = new URL(`${pathSegments.join()}/${ref.module}`, 'http://local-realm');
+    return url.pathname.slice(1);
+  }
+  return undefined;
```

Comment:
this logic drives whether to show the link to a field's card in the schema inspector. the idea was that for the base cards (like StringCard), I did not want to show a link since it's not part of the local realm. The logic here is that if the imported module's source string begins with a `.` or a `/` then we'll render a link to it. otherwise we assume its a module that is not available in the local realm.